lib/sidekiq/job_retry.rb

```diff
@@ -142,7 +142,7 @@ def attempt_retry(worker, msg, queue, exception)
 
       # App code can stuff all sorts of crazy binary data into the error message
       # that won't convert to JSON.
-      m = exception.message.to_s[0, 10_000]
```

Comment:
Hmm, I can see this causing a massive amount of headscratching. Should you provide a default error message rather than nil?

Never, ever, ever use `rescue nil`.  Be explicit and document what and why you are rescuing.

Reply:
👍 

Agreed, I don't want the retry error description to be overly confusing.

What about explicitly rescuing `StandardError` in a begin..end block and setting the default message to: `!!! ERROR MESSAGE RAISED AN ERROR !!!`?

Reply:
Yep, that's fine.

Reply:
Done 😄